code/game/objects/items/tools/mining_tools.dm

```diff
@@ -257,6 +250,11 @@
 		spark_system.set_up(5, 0, M)
 		spark_system.attach(M)
 		spark_system.start(M)
+		M.visible_message(span_danger("[src] violently vaporizes chunks of the [M] upon contact!"), visible_message_flags = COMBAT_MESSAGE)
+		if(M.stat == DEAD)
+		else
+			cell.charge += 200
+			to_chat(user, span_notice("The vaporized matter causes a temporary feedback loop in the battery! <b>Charge Amount: [cell.charge]/[cell.maxcharge]</b>"))
```

Comment:
should this not be after the return ..() because of attack_verb
Also please remove the battery charge stuff thats very spammy